.github/workflows/test-command.yml

```diff
@@ -507,6 +507,7 @@ jobs:
           export GOPATH="$HOME/go"
           export PATH="$PATH:$GOPATH/bin"
           chmod +x build/zarf
+          ./build/zarf tools registry login registry1.dso.mil --username "${{ secrets.REGISTRY1_USERNAME_ZARF_ROBOT }}" --password "${{ secrets.REGISTRY1_PASSWORD_ZARF_ROBOT }}"
```

Comment:
The e2e general-cli step is failing because it's unable to pull down a Docker image from registry1 when building the tiny-kafka package but I thought this would should have resolved that 😕

Reply:
Did you just add this line?

Reply:
Oh is this one of those cases where it has to be in master first.. Womp womp.

Reply:
GitHub Actions only look at master. If you've changed one of them you have to get that change into master before it will happen. It's definitely not ideal and something we should look more closely at, but that's the way it works right now.

Reply:
Submit a 1-line PR with just this change. We'll get it approved and merged asap